packages/core/helper-plugin/src/utils/awaitToJs.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Async await wrapper for easy error handling
+ *
+ *
+ * @deprecated This function will be removed in the next major release. Use async / await with try / catch instead.
+ */
+const to = (promise: Promise<unknown>, errorExt: object) => {
+  return promise
+    .then(function (data) {
+      return [null, data];
+    })
+    .catch(function (err) {
+      if (errorExt) {
```

Comment:
this seems optional

Reply:
@madhurisandbhor do you think we should change the code? I'm leaning more towards leaving as is even if it is redundant. Now that I think about it should errorExt be optional?

Reply:
Right, still it would be better to have correct types as we are migrating even deprecated functions to not break the functionality.

Reply:
I've updated the function so that `errorExt` is an optional parameter